### PR TITLE
Add OSX driver

### PR DESCRIPTION
**Pull request by kubukoz**

### CODE REVIEW

File: src/main/java/com/grudus/pikator2000/Window.java [hunk 1]
```diff
@@ -33,18 +32,21 @@ public void stop() throws Exception {
 
     public static void main(String[] args) {
         try {
+            String driverFileName = DriverDetector.INSTANCE.detectFileName();
+            assert(driverFileName != null); //xD
```
Comment (grudus) [inline src/main/java/com/grudus/pikator2000/Window.java]: Można fajniej - `Objects.requireNonNull(driverFileName)`

**Comment by grudus** [comment]
I don't think it was necessary to insert Kotlin here

File: src/main/java/com/grudus/pikator2000/Window.java (hunk 1 above)
Comment (kubukoz) [reply src/main/java/com/grudus/pikator2000/Window.java]: ok

**Comment by kubukoz** [comment]
Kotlin is never necessary, it's way more fun than Java though

**Pull request closed without merging**